### DISCUSSION TITLE
Δήλωση παρουσίας στο μάθημα

### DISCUSSION THREAD

**Discussion by epidrome** [courses-ionio/cscw#176] [Announcements]
Για τα ραντεβού των συναντήσεων του μαθήματος δεν θα χρησιμοποιήσουμε το calendly, αλλά τα σχόλια παρακάτω, όπου μπορείτε να δηλώνετε την παρουσία σας στην αντίστοιχη μέρα-ώρα με τα στοιχεία σας.

**Comment by epidrome** [comment]
Δευτέρα 6 Μαρτίου 6μμ

| # | ΑΜ | GitHub profile |
| -- | -- | -- |
| 1 | inf2021013 | |
| 2 | Π2016072 | |
| 3 | inf2021055 | ... |

~~Τρίτη 7 Μαρτίου 12μμ~~

| # | ΑΜ | GitHub profile |
| -- | -- | -- |
| 1 | 2003001 | gh link |
| 2 | 2003002 | gh link |
| 3 | ... | ... |

**Reply by nkanagno** [reply]
Καλησπέρα σας, θα ήθελα να δηλώσω παρουσία στο μάθημα της δευτέρας 18:00

Νικόλαος Αναγνωστόπουλος, inf2021013

**Reply by AlxiKan** [reply]
Καλησπέρα, θα ήθελα να δηλώσω παρουσία στο μάθημα της δευτέρας 18:00.

Αλέξιος Κανταρτζής Π2016072

**Reply by Axileaszervos** [reply]
Καλησπέρα σας, θα ήθελα να δηλώσω παρουσία για το μάθημα της δευτέρας, στις 18:00.

Αχιλλέας Ζερβός/inf2021055

**Comment by epidrome** [comment]
Δευτέρα 20 Μαρτίου 6μμ

| # | ΑΜ | GitHub profile |
| -- | -- | -- |
| 1 | inf2021013 |
| 2 | inf2021055 |
| 3 | Π2016072 |

**Reply by nkanagno** [reply]
Καλησπέρα σας, θα ήθελα να δηλώσω παρουσία στο μάθημα της δευτέρας 18:00

Νικόλαος Αναγνωστόπουλος, inf2021013

**Reply by Axileaszervos** [reply]
Καλησπέρα σας, θα ήθελα να δηλώσω παρουσία για το μάθημα της δευτέρας, στις 18:00.

Αχιλλέας Ζερβός/inf2021055

**Comment by AlxiKan** [comment]
Καλησπέρα, θα ήθελα να δηλώσω παρουσία στο μάθημα της δευτέρας 18:00.

Αλέξιος Κανταρτζής Π2016072

**Comment by epidrome** [comment]
Δευτέρα 3 Απριλίου 6μμ

| # | ΑΜ | GitHub profile |
| -- | -- | -- |
| 1 | inf2021013 |
| 2 | inf2021055 |
| 3 | |

**Reply by nkanagno** [reply]
Καλησπέρα σας, θα ήθελα να δηλώσω παρουσία στο μάθημα της δευτέρας 18:00

Νικόλαος Αναγνωστόπουλος, inf2021013

**Reply by Axileaszervos** [reply]
Καλησπέρα σας, θα ήθελα να δηλώσω παρουσία για το μάθημα της δευτέρας, στις 18:00.

Αχιλλέας Ζερβός/inf2021055

**Comment by epidrome** [comment]
Παρασκευή 28 Απριλίου 4μμ

| # | ΑΜ | GitHub profile |
| -- | -- | -- |
| 1 | inf2021055 |
| 2 | inf2021013 |
| 3 | |

**Reply by Axileaszervos** [reply]
Καλησπέρα σας, θα ήθελα να δηλώσω παρουσία για το μάθημα της παρασκευής, στις 16:00.

Αχιλλέας Ζερβός/inf2021055

**Reply by nkanagno** [reply]
Καλησπέρα σας, θα ήθελα να δηλώσω παρουσία στο μάθημα της παρασκευής 16:00

Νικόλαος Αναγνωστόπουλος, inf2021013

**Comment by epidrome** [comment]
Τρίτη 9 Μαΐου 12μμ 

| # | ΑΜ | GitHub profile |
| -- | -- | -- |
| 1 | Π2016072 |
| 2 | inf2021013 |
| 3 | |

**Reply by AlxiKan** [reply]
Καλησπέρα σας, θα ήθελα να δηλώσω παρουσία στο μάθημα της Τρίτης.

Αλέξιος Κανταρτζής Π2016072

**Reply by nkanagno** [reply]
Καλησπέρα σας, θα ήθελα να δηλώσω παρουσία στο μάθημα της Τρίτης 12:00

Νικόλαος Αναγνωστόπουλος, inf2021013

**Comment by epidrome** [comment]
Δευτέρα 15 Μαΐου 5μμ 

| # | ΑΜ | GitHub profile |
| -- | -- | -- |
| 1 | |
| 2 | |
| 3 | |

**Comment by epidrome** [comment]
Τρίτη 23 Μαΐου 6μμ 

| # | ΑΜ | GitHub profile |
| -- | -- | -- |
| 1 | inf2021055 |
| 2 | inf2021013 |
| 3 | |

**Reply by Axileaszervos** [reply]
Καλησπέρα σας, θα ήθελα να δηλώσω παρουσία για το μάθημα της τρίτης, στις 18:00.

Αχιλλέας Ζερβός/inf2021055

**Reply by nkanagno** [reply]
Καλησπέρα σας, θα ήθελα να δηλώσω παρουσία στο μάθημα της Τρίτης 18:00

Νικόλαος Αναγνωστόπουλος, inf2021013
